.changeset/tasty-spoons-pay.md

```diff
@@ -0,0 +1,8 @@
+---
+'@interledger/openapi': major
```

Comment:
I _think_ it should be major since it changes how `createValidatorMiddleware` is used.
But don't mind using `minor` since its just Rafiki using it

Reply:
Major sounds right, if I'm understanding correctly. This will break consumers where `NODE_ENV` is `production` and a response will fail validation.